target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8065-r7800.dts

```diff
@@ -45,6 +45,13 @@
 			pinctrl-0 = <&mdio0_pins &rgmii2_pins>;
 			pinctrl-names = "default";
 
```

Comment:
I'm to lazy to check how it was before #341. But I'm quite sure you have a good feeling how it should look like and what looks wrong.